tools/unix/generate_vulkan_shaders.sh

```diff
@@ -4,13 +4,24 @@ set -e -u
 # Prevent python from generating compiled *.pyc files
 export PYTHONDONTWRITEBYTECODE=1
 
-DEBUG="${1:-empty}"
+NDK_PATH="${1:-empty}"
+DEBUG="${2:-empty}"
 
 MY_PATH="`dirname \"$0\"`"              # relative
 MY_PATH="`( cd \"$MY_PATH\" && pwd )`"  # absolutized and normalized
 
-source "$MY_PATH/../autobuild/ndk_helper.sh"
-export NDK_ROOT=$(GetNdkRoot) || ( echo "Can't read NDK root path from android/local.properties"; exit 1 )
+if [ -z "$NDK_PATH" ]
```

Comment:
1. It's better to avoid similar variables and just use NDK_ROOT everywhere.
2. Can NDK_ROOT variable be checked in the ndk_helper.sh and returned from there?
3. Most installations of Android Studio define ANDROID_HOME variable. Can ndk_helper be updated to extract the NDK path from there?
`$ANDROID_HOME/ndk/25.2.9519653` for example? `ls -1 ${ANDROID_HOME/ndk} | sort | tail -1`

Reply:
Probably, this line `export NDK_ROOT=$(GetNdkRoot)` returns an empty string in my environment. I'll try to use your proposal with ANDROID_HOME

Reply:
Done